python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/dhcp_servers.schema.yml

```
@@ -170,6 +170,25 @@ keys:
                       type: str
                       description: |-
                         Valid IPv4 address from the given subnet.
+              tftp_server:
+                type: dict
+                keys:
+                  file_ipv4:
```

we are under ipv4_subnets so no need to complexify the key name :) (same for other keys)

also let's call it `file` and make this a string
  

```suggestion
                  file:
```

---

```
localhost(config-dhcp-subnet-ipv4)#tftp server file BLAH BLOH BLIG BLLLLLE
localhost(config-dhcp-subnet-ipv4)#show active
dhcp server
   subnet 23.23.23.0/24
      tftp server file BLAH BLOH BLIG BLLLLLE

```

---

@ernestoherrerab does it work really to put multiple files?

the show output shows it in singular:

```
ocalhost(config-dhcp-subnet-ipv4)#show dhcp server ipv4
IPv4 DHCP server is inactive
Client ID echo: enabled
Lease duration: 0 days 2 hours 0 minutes
TFTP file: jhshshsh sjajajsj alslaskal
Active leases: 0
IPv4 DHCP interface status:
   Interface    Status
--------------- ------


Subnet: 23.23.23.0/24
Subnet name:
DNS server(s):
Default gateway address: 0.0.0.0
TFTP boot file: BLAH BLOH BLIG BLLLLLE
Active leases: 0
```

---

So I asked @ClausHolbechArista  and when the keys were introducedfor top level `dhcp server` the choice was made to have these as a simple string: https://github.com/aristanetworks/avd/pull/3993#discussion_r1607711648

please do the same here

EOS show commands seems to indicate only one file (singular).

Feel free to comment if you have concern / usage of multiple string crammed in the same field?